File: CPP/Program/Course.cpp
#include "Header.hh"

Course::Course()
{
    nama_matakuliah = "";
}

Course::Course(string nama_matakuliah)
{
    this->nama_matakuliah = nama_matakuliah;
}

string Course::getNama_matakuliah()
{
    return nama_matakuliah;
}

void Course ::setNama_matakuliah(string nama_matakuliah)
{
    this->nama_matakuliah = nama_matakuliah;
}

Course::~Course()
{

}

File: PYTHON/Program/Course.py
class Course:
    __nama_matakuliah = ""

    def __init__(self, nama_matakuliah = ""):
        # Set attributes pada class mahasiswa
        self.__nama_matakuliah = nama_matakuliah

    # Getter dan Setter

    # Getter dan setter untuk nama mahasiswa
    def get_nama_matakuliah(self):
        return self.__nama_matakuliah
    def set_nama_matakuliah(self, nama_matakuliah):
        self.__nama_matakuliah = nama_matakuliah